template/.github/workflows/test.yml

```diff
@@ -28,7 +28,16 @@ jobs:
         run: npm run lint
 
       - name: Run unit tests
-        run: npm run test
+        run: npm test -- --coverage
 
       - name: Run integration tests
         run: npm run cypress
+
+      - name: Merge code coverage reports
+        run: npm run test:merge-coverage
+
+      - name: Danger
+        run: npx danger ci
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DANGER_GITHUB_API_TOKEN: ${{ secrets.DANGER_GITHUB_API_TOKEN }}
```

Comment:
Can we use the same `GITHUB_TOKEN`?

Reply:
Nice catch! The `DANGER_GITHUB_API_TOKEN` was not needed (it was used in a previous test I did).
thanks!!! 🙌 

Fixed ;-)